code/src/java/pcgen/persistence/PersistenceManager.java

```diff
@@ -65,7 +65,7 @@ public static PersistenceManager getInstance()
 	 * 
 	 * CODE-1889 to remove use of this method
 	 */
-	public void setChosenCampaignSourcefiles(List<URI> l)
+	public static void setChosenCampaignSourcefiles(List<URI> l)
```

Comment:
This is now a completely confusing class design.  You still have a getInstance() method, but it returns an instance with no non-static methods?  I know the original design was to have different managers, but that design isn't being pursued, but if you are going to get rid of non-static methods, then getInstance() makes no sense.